### PR TITLE
Adding lockfiles to examples

### PR DESCRIPTION
**Pull request by BrunoWilkinson**
My attempt to this issue #94 (it's a bit hacky in my opinion)😄 

Solution:

- adding `"lock-file": "yarn generate-lock-entry >> yarn.lock"` to the `package.json` for every example projects
- adding `"example:generate-lock-file": "lerna run --scope @examples/* lock-file"` to the root `package.json`
- run `yarn example:generate-lock-file`, all the example project should have a lock file

### CODE REVIEW

File: examples/next-chat/yarn.lock [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    jest-playwright-preset "^1.4.5"
+    next "^10.0.5"
+    playwright "^1.8.0"
+    react "^17.0.1"
+    react-dom "^17.0.1"
+    react-query "^3.6.0"
+    start-server-and-test "^1.12.0"
+    superjson "^1.5.2"
+    zod "^3.0.0"
+
```
Comment (KATT) [inline examples/next-chat/yarn.lock:24]: This isn't a "proper" lockfile unfortunately, it just regurgitates what's in `package.json`, doesn't contain hashes of the "current" version, etc

**Review by KATT — CHANGES_REQUESTED** [review]
Thanks for the PR, feel free to have another go, it's a tricky one :) 

I had a working version with `npm`'s `package-lock.json` in #100 but I don't wanna mix yarn and npm

File: examples/next-chat/yarn.lock (hunk 1 above)
Comment (BrunoWilkinson) [reply examples/next-chat/yarn.lock:24]: That's a shame, using this yarn command to generate:
https://classic.yarnpkg.com/en/docs/cli/generate-lock-entry/

I'm thinking that maybe the example shouldn't part of yarn workspace, so we can generate the yarn lock for each.

I will try and give another go this week, hopefully :)

**Comment by KATT** [comment]
Closing due to inactivity

**Comment by BrunoWilkinson** [comment]
@KATT Apologies for the inactivity, too busy because of work.

**Comment by KATT** [comment]
No problem @BrunoWilkinson - feel free to pick it up later!

**Pull request closed without merging**